openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_ESC_variables.py

```diff
@@ -312,3 +312,159 @@ class HVAC1_PDRS__postcode(Variable):
+        AC_HSPF_cold = buildings('HVAC1_HSPF_cold', period)
+        AC_ACOP = buildings('HVAC1_rated_ACOP_input', period)
+        product_class = buildings('HVAC1_Air_Conditioner_type', period)
+        new_AC_heating_capacity = buildings('HVAC1_heating_capacity_input', period)
+        heating_capacity = np.select(
+                                    [
+                                        (new_AC_heating_capacity < 4),
+                                        ((new_AC_heating_capacity >= 4) * (new_AC_heating_capacity < 6)),
+                                        ((new_AC_heating_capacity >= 6) * (new_AC_heating_capacity < 10)),
+                                        ((new_AC_heating_capacity >= 10) * (new_AC_heating_capacity < 13)),
+                                        ((new_AC_heating_capacity >= 13) * (new_AC_heating_capacity < 25)),
+                                        ((new_AC_heating_capacity >= 25) * (new_AC_heating_capacity <= 65)),
+                                        (new_AC_heating_capacity > 65)
+                                    ],
+                                    [
+                                        "less_than_4kW",
+                                        "4kW_to_6kW",
+                                        "6kW_to_10kW",
+                                        "10kW_to_13kW",
+                                        "13kW_to_25kW",
+                                        "25kW_to_65kW",
+                                        "over_65kW"
+                                    ]
+                                    )
+
+        climate_zone = buildings('HVAC1_certificate_climate_zone', period)
+        climate_zone_str = np.select([climate_zone == 1, climate_zone == 2, climate_zone == 3],
+                                     ['hot_zone', 'average_zone', 'cold_zone'])
+
+        in_hot_zone = (climate_zone_str == 'hot_zone')
+        in_average_zone = (climate_zone_str == 'average_zone')
+        in_cold_zone = (climate_zone_str == 'cold_zone')
+
+        AC_HSPF = np.where(
+                            in_cold_zone,
+                            AC_HSPF_cold,
+                            AC_HSPF_mixed)
+        # determines which HSPF value to use
+        HSPF_is_zero = (
+                        (AC_HSPF == 0) + 
+                        (AC_HSPF == None)
+                        )
+        # tells you if the relevant HSPF is zero or non-existant
+        AC_exceeds_benchmark = np.select([
+                                            HSPF_is_zero,
+                                            np.logical_not(HSPF_is_zero) * in_cold_zone,
+                                            np.logical_not(HSPF_is_zero) * np.logical_not(in_cold_zone),
+                                            ],
+                                            [
+            (AC_ACOP >= parameters(period).ESS.HEER.table_D16_5['ACOP'][product_class][heating_capacity]),
```

Comment:
I believe all three of these values should be pulling [cooling_capacity] not heatimg_capacity, although heating capacity is coded for this, the rules uses cooling capacity for all of these D16.4, D16.5 tables.